docs/UserGuide.md

```diff
@@ -139,63 +138,138 @@ Exits the program.
+* Business Administration (Accountancy)
+* Business Analytics 
+* Chemical Engineering 
+* Chemistry
+* Chinese Language
+* Chinese Studies 
+* Communications and New Media 
+* Civil Engineering 
+* Computer Engineering 
+* Computer Science 
+* Data Science and Analytics 
+* Data Science and Economics 
+* Dentistry 
+* Economics 
+* Electrical Engineering 
+* English Language 
+* English Literature 
+* Environmental Engineering 
+* Environmental Studies 
+* Food Science and Technology 
+* Geography 
+* Global Studies 
+* History 
+* Japanese Studies 
+* Industrial Design 
+* Industrial Engineering 
+* Industrial and Systems Engineering 
+* Information Systems 
+* Information Security 
+* Infrastructure and Project Management 
+* Landscape Architecture 
+* Law 
+* Life Sciences 
+* Malay Studies 
+* Management 
+* Marketing 
+* Materials Science and Engineering 
+* Mathematics 
+* Mechanical Engineering 
+* Medicine
+* Pharmaceutical Science 
+* Philosophy 
+* Philosophy, Politics, and Economics 
+* Physics 
+* Political Science 
+* Psychology 
+* Quantitative Finance 
+* Real Estate 
+* Social Work 
+* Sociology 
+* South Asian Studies 
+* Southeast Asian Studies 
+* Statistics 
+* Systems Engineering 
+* Theatre Studies 
+* Urban Studies 
+* Visual Communications
+
+If your major is not on this list, you can simply write "Others" as your major.<br>
+
+If you have a double major or a major-minor, you are required to choose only one major. Please select any one of the majors listed above.
 
 #### Expected Output (Success):
 * GUI: New student entry added in the main student list
 * Message: “Details added successfully! New student added: (details of new student)”
-  ![Add feature](images/add.png)
+![Add feature](images/add.png)
 
-* Social Media Link can be clicked
+Note: 
+* Social media links can be clicked
 * The link will be opened on a browser upon clicking
 
 ![social_media](images/socialMediaLinks.png)
-<div style="text-align: center;"> Output with single social media link
+<div style="text-align: center;">Output with a single social media link</div>
 
 ![social_media](images/multipleLinks.png)
-Output with multiple social media links</div>
+<div style="text-align: center;">Output with multiple social media links</div>
 
 #### Expected Output (Failure):
-* Message:  "Invalid command format! add: Adds a person to StudentConnect. <br>
-  Parameters: n/NAME m/MAJOR y/YEAR e/EMAIL d/DESCRIPTION t/TUTORIALS sm/SOCIAL_MEDIA_LINK nt/NATIONALITY g/GENDER<br>
+* Message:  "Invalid command format! add: Adds a student to StudentConnect. <br>
+  Parameters: n/NAME m/MAJOR y/YEAR e/EMAIL d/DESCRIPTION [t/TUTORIALS]... [sm/SOCIAL_MEDIA_LINK]... nt/NATIONALITY g/GENDER<br>
   Example: add n/John Doe m/Computer Science y/2 e/johnd@u.nus.edu d/I love programming in my free time<br>
-  t/02 17 20 sm/https://www.linkedin.com/in/john-doe-123456789 nt/local g/m";
-
+  t/02 sm/https://www.linkedin.com/in/john-doe-123456789 nt/local g/M";
```

Comment:
is this supposed to be under failure I'm a bit confused

Reply:
Yes. This is the message for invalid command format, so the example here is a valid e.g. that people can use.

Reply:
Okay got it thanks!